srcpkgs/rooster/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'rooster'
 pkgname=rooster
-version=2.8.6
+version=2.9.0
 revision=1
 build_style=cargo
 hostmakedepends="pkg-config python3"
-makedepends="libX11-devel libXmu-devel libressl-devel libsodium-devel"
+makedepends="libX11-devel libXmu-devel libressl-devel libsodium-devel libssl48"
 short_desc="Simple password manager for geeks"
 maintainer="Ulf <void@uw.anonaddy.com>"
 license="Apache-2.0"
 homepage="https://github.com/conradkdotcom/rooster"
 distfiles="https://github.com/conradkdotcom/rooster/archive/v${version}.tar.gz"
-checksum=b05d2c368ac26d96edd058130bfc7cc7a1960a5bfbc63b6ddfad62d27b08d62f
+checksum=2cb08c90c9604951c810038dffd1dfca1de71bfe1d76b22f951dc3f14c0efab4
 
 pre_build() {
-	cargo update --package openssl-sys --precise 0.9.58
+	cargo update --package openssl-sys
```

Comment:
Either completely remove this or leave the precise update.

Reply:
Couldn't get it to compile with that flag before. Same goes for libssl. Compiled fine now though.